lib/puppet/type/f5_profileclientssl.rb

```diff
@@ -0,0 +1,71 @@
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/parameter/f5_name.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_address.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_availability_requirement.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_connection_limit.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_connection_rate_limit.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_description.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_health_monitors.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_ratio.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_state.rb'))
+
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_truthy.rb'))
+
+Puppet::Type.newtype(:f5_profileclientssl) do
+  @doc = 'Manage Client SSL profile objects'
+
+  apply_to_device
+  ensurable
+
+  newparam(:name, :parent => Puppet::Parameter::F5Name, :namevar => true)
+
+  newproperty(:description, :parent => Puppet::Property::F5Description)
+
+  newproperty(:cert) do
+    desc "cert"
+  end
+
+  newproperty(:key) do
+    desc "key"
+  end
+
+  newproperty(:proxy_ssl, :parent => Puppet::Property::F5truthy) do
+    desc "Valid values are 'enabled' or 'disabled'."
+    truthy_property('Fail Safe')
+  end
+
+  newproperty(:proxy_ssl_passthrough, :parent => Puppet::Property::F5truthy) do
+    desc "Valid values are 'enabled' or 'disabled'."
+    truthy_property('Fail Safe')
+  end
+
+
+  newproperty(:ssl_forward_proxy, :parent => Puppet::Property::F5truthy) do
+    desc "Valid values are 'enabled' or 'disabled'."
+    truthy_property('Fail Safe')
+  end
+
+  newproperty(:ssl_forward_proxy_bypass, :parent => Puppet::Property::F5truthy) do
+    desc "Valid values are 'enabled' or 'disabled'."
+    truthy_property('Fail Safe')
+  end
+
+  newproperty(:peer_cert_mode) do
+    desc "peer_cert_mode."
+    newvalues(:'ignore', :'require')
```

Comment:
Quoting rubobop...
> Do not use strings for word-like symbol literals

Reply:
@ericzji please change 'ignore' and 'require' to :ignore and :require accordingly.